sh/tests/to-git-or-not-to-git_test.sh

```diff
@@ -4,14 +4,18 @@
 set -euo pipefail
 IFS='
 '
+
 FILENAME="student/to-git-or-not-to-git.sh"
+submitted=$(bash student/to-git-or-not-to-git.sh)
```

Comment:
We declare `FILENAME`, but then don't use it here.

Reply:
fixed it